src/components/types.ts

```diff
@@ -31,6 +31,8 @@ export interface InputProps extends InputStatus, InputHTMLAttributes<HTMLInputEl
    * Slot for an icon on the right
    */
   endIcon?: ReactNode;
+
+  weight?: number | undefined;
```

Comment:
Don't think the undefined is needed here as `?:` indicates this is an optional param